stdlib/src/builtin/string.mojo

```diff
@@ -400,7 +400,22 @@ struct String(Sized, Stringable, IntableRaising, KeyElement, Boolable):
         self = str(value)
 
     @always_inline
-    fn __init__(inout self, ptr: Pointer[Int8], len: Int):
+    fn __init__(inout self, ptr: UnsafePointer[Int8], len: Int):
```

Comment:
**Suggestion** In the future, given how unsafe both this constructor and the pre-existing one below it are, it would be good to make these keyword-arguments.  I filed an issue internally about this sort of constructor for `StringRef` last month and Chris said:

> Driveby comment: if you do this, please make sure to require a keyword argument since it will be very unsafe.